components/suggest/android/build.gradle

```diff
@@ -5,6 +5,6 @@ dependencies {
     api project(":remotesettings")
 }
 
-ext.configureUniFFIBindgen("../src/suggest.udl")
+ext.configureUniFFIBindgen("suggest")
```

Comment:
Thanks so much for migrating Suggest over, too!

Does its `uniffi.toml` need to change, too, to remove `cdylib_name` (and the `external_packages`, it looks like? 👀)

Reply:
That makes sense to me, I just made that update.  Is there anything I should do to test if it's still working properly?